### PR TITLE
Skip test_fake_crossref_backward_amp_nn_functional_bilinear_xpu_float32

### PR DESCRIPTION
**Pull request by jkosnox**
test_fake_crossref_backward_amp_nn_functional_bilinear_xpu_float32 fails with a FakeTensor metadata mismatch on aten.view.default (outputs_alias_inputs: False != True). This is a known upstream PyTorch bug triggered by zero-element tensors in nn.functional.bilinear's backward pass — the same test is also disabled for CUDA ([#159151](https://github.com/pytorch/pytorch/issues/159151), [#159150](https://github.com/pytorch/pytorch/issues/159150))

Fixes:
[#2250](https://github.com/intel/torch-xpu-ops/issues/2250)

### CODE REVIEW

**Comment by BBBela** [comment]
The PyTorch issues mentioned in the description skip two similar test cases
(https://github.com/pytorch/pytorch/issues/159151, https://github.com/pytorch/pytorch/issues/159150)
`test_fake_crossref_backward_amp_nn_functional_bilinear_xpu_float32`
and
`test_fake_crossref_backward_no_amp_nn_functional_bilinear_xpu_float32`

I wanted to ask why this PR adds skip for only one of them instead of both of them while both are mentioned in the description? And the other one is failing in the CI.

If the root cause and the problem is the same, maybe we could skip both those test cases?

**Comment by jkosnox** [comment]
> The PyTorch issues mentioned in the description skip two similar test cases ([pytorch/pytorch#159151](https://github.com/pytorch/pytorch/issues/159151), [pytorch/pytorch#159150](https://github.com/pytorch/pytorch/issues/159150)) `test_fake_crossref_backward_amp_nn_functional_bilinear_xpu_float32` and `test_fake_crossref_backward_no_amp_nn_functional_bilinear_xpu_float32`
> 
> I wanted to ask why this PR adds skip for only one of them instead of both of them while both are mentioned in the description? And the other one is failing in the CI.
> 
> If the root cause and the problem is the same, maybe we could skip both those test cases?

Makes sense, done

**Comment by BBBela** [comment]
I suppose we can add the `disable_all` label not to overuse CI resources.
All the [PREVIOUS CHECKS](https://github.com/intel/torch-xpu-ops/pull/3403/checks?sha=81d321a864e441f60114a559bc55f1174f1132a9) on this PR passed. The only failing one was the one that is currently added to the skip list.
So the only change since [LAST CI RUN](https://github.com/intel/torch-xpu-ops/pull/3403/checks?sha=81d321a864e441f60114a559bc55f1174f1132a9) is adding new, but similar test case to skip list. I believe it does not require to launch all the tests again.



But please re-enable the CI checks if any new functional change is added to this PR. Thanks!

**Review by BBBela — APPROVED** [review]
Looks good to me. 
Both tests have upstream PyTorch issues and are disabled for CUDA, so it seems reasonable to skip them also for XPU until they are fixed in the upstream.
Thanks!

File: test/xpu/skip_list_common.py [hunk 1]
```diff
@@ -128,6 +128,11 @@
         # For CUDA it's skipped explicitly in common_methods_invocations.py in upstream. We can skip it here
         "test_out_histc_xpu_float32",
         "test_out_mean_xpu_float32",
+        # FakeTensor mismatch in outputs_alias_inputs for aten.view.default
+        # Known upstream issue: https://github.com/pytorch/pytorch/issues/159150
+        "test_fake_crossref_backward_amp_nn_functional_bilinear_xpu_float32",
+        # Known upstream issue: https://github.com/pytorch/pytorch/issues/159151
+        "test_fake_crossref_backward_no_amp_nn_functional_bilinear_xpu_float32",
```
[the remark this answers is not in the document]
Comment (jkosnox) [reply test/xpu/skip_list_common.py:135]: Ok

**Review by pbielak — APPROVED** [review]

**Comment by chuanqi129** [comment]
✅ PR has been successfully merged by @jkosnox.
